src/Env.php

```diff
@@ -44,4 +46,54 @@ public static function get($name, $default = null)
             return $originalValue;
         }
     }
+    
+    /**
+     * 
+     * @param string $varname
+     * @return string
+     * @throws EnvVarNotFoundException
+     */
+    public static function requireEnv($varname)
+    {
+        $value = self::get($varname);
+        
+        if ($value === null) {
+            $message = 'Required environment variable: ' . $varname . ', not found.';
```

Comment:
Your suggestion is the way I had it first, but Scrutinizer didn't like the use of interpolation, so I changed it to this.